### ISSUE TITLE
unable to fork from github to azure repos

### ISSUE THREAD

**Issue by robidas** [MicrosoftDocs/azure-devops-docs#13636]
In the first section or this tutorial, Get the code,  the instruction is to fork the sample app in Github or Azure Repos. I need to use Azure Repos. But there's no option in Azure Repos to fork from Github. The only way to get the code from Github into azure repos appears to be to import or clone. I don't think this is the same as "fork". A fork is different than a clone or an import. It maintains a relationships to the original code that was forked. I don't know why forking is necessary for this tutorial, rather than importing or cloning, but I'm just trying to follow the example. Could you please provide an explanation how to fork the Github repository to Azure Repos? or should I just clone instead of fork? Any help would be appreciated. Thanks.
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 4cfc7aa5-d9ff-ad64-d26e-d2bd10470025
* Version Independent ID: 9b4f02ea-dab9-f613-5345-496435ab2408
* Content: [Deploy a web app to Azure Web App for Containers - Azure Pipelines](https://learn.microsoft.com/en-us/azure/devops/pipelines/apps/cd/deploy-docker-webapp?view=azure-devops&tabs=dotnet-core%2Cyaml)
* Content Source: [docs/pipelines/apps/cd/deploy-docker-webapp.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/main/docs/pipelines/apps/cd/deploy-docker-webapp.md)
* Service: **azure-devops-pipelines**
* Sub-service: **azure-devops-pipelines-apps**
* GitHub Login: @juliakm
* Microsoft Alias: **jukullam**

**Comment by WilliamAntonRohm** [comment]
@robidas -- Rob, thank you for your feedback.  Please consider these resources:
- [Azure DevOps Services](https://azure.microsoft.com/support/devops/) 
- [Azure DevOps on Stack Overflow](https://stackoverflow.com/questions/tagged/azure-devops)

@juliakm -- Julia, please look into this issue.

**Comment by juliakm** [comment]
@robidas You're right, you can't fork in Azure Repos from GitHub. You'll need to import the repository instead. Here's an [overview doc](https://learn.microsoft.com/en-us/azure/devops/repos/git/import-git-repository?view=azure-devops). 

I'll update the article to make this clearer.
